File: Backend/Employee.CURD.Model/EmployeeModel.cs
using System.ComponentModel.DataAnnotations;

namespace Employee.CURD.Model
{
    public class EmployeeModel
    {
        public Guid Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Mobile { get; set; }
        public long Salary { get; set; }
        public string Role { get; set; }

    }

}

File: Frontend/src/app/employee.model.ts
export class Employee {
  id: string = '';
  firstName: string = '';
  lastName: string = '';
  email: string = '';
  mobile: string = '';
  salary: number = 0;
  role: string = '';
  
}
export class PagingEmployee {
  employees = [];
  totalCount = 0;
  pageIndex = 0;
  pageSize = 0;
}
